src/main/webapp/tests/mocks/models/mockDepositLocation.js

```diff
@@ -94,6 +94,17 @@ var mockDepositLocation = function($q) {
         return payloadPromise($q.defer(), testConnectionPayload);
     };
 
+    model.testDepositLocation = function() {
```

Comment:
Just to note, these methods are augmented methods and do not belonging to the model/prototype. They require the controller scope and hence violating object-oriented principles of loose coupling. For cleaner code, they should be moved to the model and add arguments (monadic or dyadic preferred) to avoid this. Therefore, the unit tests would be against the model for these and mocked for the controller.

Reply:
This is addressed here:
- https://github.com/TexasDigitalLibrary/Vireo/pull/1818